### ISSUE TITLE
fatal error: CGAL/optimal_bounding_box.h: No such file or directory  #include <CGAL/optimal_bounding_box.h>

### ISSUE THREAD

**Issue by Zerg-Overmind** [CGAL/cgal#7012]
_Please use the following template to help us solving your issue._

## Issue Details

I was trying to use optimal_bounding_box in python, and so I added 

add_library(cgal_test SHARED cgal_cpp_test.cpp)

in my local CMakeLists.txt to output cgal_test.so file. Under ./build/debug I typed

make

And then I got the error like:

fatal error: CGAL/optimal_bounding_box.h: No such file or directory #include <CGAL/optimal_bounding_box.h>

I really didn't know what happened. Do you know how to generate .so together with executable file?

**(BTW, is this because cgal is a header-only library and I need to explicitly run "make install" to install cgal header files?)**

**Comment by sloriot** [comment]
More details are needed. CGAL is a c++ library but you mention python. Can you give more details about what you did?

**Comment by Zerg-Overmind** [comment]
Sry for the late reply, the problem has been solved! 
Actually I want to use the function "optimal bounding box" or "oriented bounding box" in a python project so I wrote a function in c++ and compiled it (got excutable file and .so file) for python. And I got the error mentioned above during compiling. 
I noticed that this cgal is defaultly a header-only project and I additionally ran:

cmake .
make install

to install all the header files referring to https://doc.cgal.org/latest/Manual/usage.html
